engine/src/fields.cpp

```diff
@@ -477,7 +477,7 @@ void MCField::setparagraphs(MCParagraph *newpgptr, uint4 parid, findex_t p_start
         else
             t_lastpgptr = newpgptr -> prev();
 
-        t_insert_paragraph->setselectionindex(p_start, p_start, False, False);
+        t_insert_paragraph->setselectionindex(PARAGRAPH_MAX_LEN, PARAGRAPH_MAX_LEN, False, False);
```

Comment:
The indextoparagraph() method converts the input start / end into paragraph relative indicies - relative to the paragraph passed in... So, assuming that it is 'correct' to set the selection index to p_start in the paragraph containing it then the previous code was already correct, and this change is actually incorrect.